infrastructure/src/main/java/org/corfudb/infrastructure/log/StreamLogFiles.java

```diff
@@ -119,6 +120,21 @@ public StreamLogFiles(ServerContext serverContext, boolean noVerify) {
         }
     }
 
+    /**
+     * Create stream log directory if not exists
+     */
+    private void initStreamLogDirectory() {
+        if (logDir.toFile().exists()) {
```

Comment:
Maybe log the fact the it already existed.

Reply:
done